src/openforms/submissions/models.py

```diff
@@ -265,6 +272,41 @@ def get_merged_data_with_component_type(self) -> dict:
 
         return merged_data
 
+    def get_merged_appointment_data(self) -> dict:
```

Comment:
I'd prefer to see detailed type hints in the form of `Dict[str, ???]` so I know what types the key/values have.